db/schema.rb

```diff
@@ -150,13 +150,16 @@
   create_table "people", force: :cascade do |t|
     t.string   "email"
     t.string   "phone"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
     t.string   "first_name"
     t.string   "last_name"
+    t.string   "uuid"
+    t.boolean  "is_volunteer"
   end
 
   add_index "people", ["email"], name: "index_people_on_email", unique: true, using: :btree
+  add_index "people", ["uuid"], name: "index_people_on_uuid", unique: true, using: :btree
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.